distribution/scripts/ciphertool.sh

```
@@ -0,0 +1,122 @@
+  JAVA_HOME=`cygpath --absolute --windows "$JAVA_HOME"`
+  ICP_HOME=`cygpath --absolute --windows "$ICP_HOME"`
+  CLASSPATH=`cygpath --path --windows "$CLASSPATH"`
+fi
+
+# ----- Execute The Requested Command -----------------------------------------
+$JAVA_HOME/bin/java $JAVA_OPTS \
+  -Dcarbon.home="$ICP_HOME" \
+  -Dcarbon.config.dir.path="$ICP_HOME/conf" \
+  -Dorg.wso2.CipherTransformation="RSA/ECB/OAEPwithSHA1andMGF1Padding" \
+  -classpath "$ICP_CLASSPATH" \
+  org.wso2.ciphertool.CipherTool $*
```

Shellcheck (0.11.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)